src/App.js

```diff
@@ -13,7 +13,7 @@ function App() {
     <div className="App">
       <Router>
         <header className="App-header">
-          <h1>Bookstore</h1>
+          <h1>Bookstore CMS</h1>
           <NavMenu />
         </header>
         <Routes>
```

Comment:
- [ ] Please, consider that some points are not cover the requested styling. Here on this screenshot, I have marked some points to be fixed.

1. Try to move and align the section with the header "Bookstore CMS" and links to the right, to be on the same line with the books list 👍 
2. Please, try to insert the profile icon where I marked it, as is shown on the original design.
3. Please, consider fixing the styling of the buttons - they should be without the border and fixed in the needed color.
4. "Books" and "Categories" should be uppercase
5. Please, try to insert a divider between "Comments", "Remove" and "Edit"

>Your design
<img width="1423" alt="Screen Shot 2022-03-30 at 23 27 04" src="https://user-images.githubusercontent.com/49064106/160917764-b025a4ee-f4a7-438a-83c6-05ffde253b14.png">

> The requested styling:
<img width="742" alt="Screen Shot 2022-03-30 at 23 26 51" src="https://user-images.githubusercontent.com/49064106/160917863-aebc5db1-9b93-4a0e-b3b9-6cd65dd2fbdf.png">